charts/node/templates/statefulset.yaml

```diff
@@ -288,6 +289,28 @@ spec:
           securityContext:
             runAsUser: 0
         {{- end }}
+        {{- if .Values.node.persistGeneratedNodeKey }}
+        - name: persist-generated-node-key
```

Comment:
how it was generated before?

Reply:
I guess the node key is generated when the node starts. So we did not need to explicitly generate it. Not sure how `persistGeneratedNodeKey` worked though

Reply:
Not all implimetation of substrate node will have `key` option, we may need to create separate container  with `subkey`
```
error: Found argument 'key' which wasn't expected, or isn't valid in this context

USAGE:
    malus dispute-ancestor [OPTIONS]

For more information try --help
```

Reply:
Yes but this is inconvenient as there are no official binary releases or docker image of Subkey.

Reply:
`persistGeneratedNodeKey` was broken for a few releases as the code disappeared during a previous refactor.

Reply:
we can try to use this docker image, it even had update 1 hour ago: 
https://hub.docker.com/r/parity/subkey/tags